starboard/rdk/gyp_configuration.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2017 The Cobalt Authors. All Rights Reserved.
```

Comment:
Can you update the copyright year to 2023? For a new file it should have the current year.